pkg/util/util.go

```diff
@@ -47,3 +50,29 @@ func Print(format string, in interface{}, w io.Writer, fn PrintTable) error {
 	_, err = fmt.Fprint(w, string(out))
 	return err
 }
+
+// Sanitize and validate HTTP listen address.
+// It accepts "port", ":port", "ip:port", "[ipv6]:port" as valid addresses
+// if only "port" is provided, the semi-colon prefix is added
+func SanitizeAddr(addr string) (string, error) {
+	if !strings.Contains(addr, ":") {
+		addr = ":" + addr
+	}
+
+	ip, port, err := net.SplitHostPort(addr)
+	if err != nil {
+		return "", err
+	}
+
+	if ip != "" {
+		if i := net.ParseIP(ip); i == nil {
+			return "", fmt.Errorf("invalid IP address: %s", ip)
```

Comment:
```suggestion
			return "", fmt.Errorf("invalid IP address: %q", ip)
```

to put quotes around the string